packages/patternfly-4/react-core/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { HTMLProps } from 'react';
+import styles from '@patternfly/patternfly/components/FormControl/form-control.css';
```

Comment:
With new styles will have to change to `@patternfly/react-styles/components/FormControl/form-control`